docs/source/contributing/index.rst

```diff
@@ -20,4 +20,5 @@ Contribution Guidelines
     documentation.rst
     release.rst
     checklist.rst
+    pull_request.rst
```

Comment:
The contribution checklist should be the last item in the contribution documentation. Logically, how to create a pull request should be explained before the contribution is submitted. I would like to see a reference to the pull request documentation in the checklist.rst file.